version/distro/distro_test.go

```diff
@@ -13,3 +13,26 @@ func BenchmarkGet(b *testing.B) {
 	}
 	_ = d
 }
+
+func TestIsUDMPro(t *testing.T) {
```

Comment:
I just realized this doesn't actually test anything 🤦. I've mostly written Ruby where you can stub file system calls to unit test these sorts of things. What is the right pattern in go?

Reply:
Would it be acceptable to introduce something like https://github.com/spf13/afero so I can test the `isUDMPro()` function?

Reply:
That seems like maybe more complexity than we need here.

In this case, what does the test really tell us? We're checking that the strings we probed for matched some files in the test, but what really matters is what those files contain on the real device. I think mostly we're checking is whether the code changed.

On that basis I'd probably skip this test. But if we do want to do it, I guess we could shim in a `fs.StatFS` and rework the helpers to use it.  I'd prefer to avoid pulling in that whole third-party library without a stronger need for it. @DentonGentry what do you think?

Reply:
Yeah, I agree. The value of this test is relatively low given the likely hood of change here. I think we might add/remove checks, but I can't imagine it'll ever be complicated. And if it is, we can always add a test at that time.